distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/opensearch/OpenSearchTestCommons.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.common.test.opensearch;
+
+import java.util.HashMap;
+
+import ddf.common.test.ServiceManager;
+
+public class OpenSearchTestCommons {
+
+    public static ServiceManager serviceManager;
+
+    public static class OpenSearchSourceProperties extends HashMap<String, Object> {
+
+        public static final String SYMBOLIC_NAME = "catalog-opensearch-source";
+
+        public static final String FACTORY_PID = "OpenSearchSource";
+
+        public OpenSearchSourceProperties(String sourceId, String openSearchUrl, ServiceManager serviceManager) {
```

Comment:
Could this simply be a top level class? Or maybe we move away from this whole concept of extending from `HashMap` and just have a method that takes in the `sourceId` and `openSearchUrl` and returns a `HashMap`?